firefox-ios/firefox-ios-tests/Tests/XCUITests/MicrosurveyTests.swift

```diff
@@ -20,6 +20,23 @@ final class MicrosurveyTests: BaseTestCase {
         XCTAssertTrue(app.buttons[AccessibilityIdentifiers.Microsurvey.Prompt.closeButton].exists)
     }
 
+    func testURLBorderHidden_whenMicrosurveyPromptShownAndAppearsWhenClosed() throws {
```

Comment:
only comment would be that the underscore should be removed and the title should be smaller

Reply:
but can be changed some other time

Reply:
updated! I'll change it while we here and have some time to get this merge in before next nightly. Thanks for calling this out!